src/components/theme/App/App.jsx

```diff
@@ -177,6 +182,7 @@ class App extends Component {
 export const __test__ = connect(
   (state, props) => ({
     pathname: props.location.pathname,
+    token: state.userSession.token,
```

Comment:
@nzambello @pnicolli any idea how this got in here? I just merged master into this PR branch and I think this started to show up here.

Reply:
That's for `is-anonymous` and `is-authenticated` CSS classes (https://github.com/plone/volto/blame/93b77e4d6a68778e739628aac96681d6dff97816/src/components/theme/App/App.jsx#L137).
Looking at the diff, it was missing in test connect.